### PR TITLE
[TIMOB-25269] iOS: Use final iPhone X launch-image names

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25269

### CODE REVIEW

**Comment by hansemannn** [comment]
@sgtcoolguy Blocked here. Project-related logic-changes, so no new unit-tests applicable.

**Comment by sgtcoolguy** [comment]
@hansemannn seems ok to me to merge without tests... But we need QE to test first.

**Review by sgtcoolguy — APPROVED** [review]

**Comment by hansemannn** [comment]
Yep, @ewieberappc is on it, thanks! 🙂

File: iphone/cli/commands/_build.js [hunk 1]
```diff
@@ -3959,7 +3959,7 @@ iOSBuilder.prototype.writeInfoPlist = function writeInfoPlist() {
 		'name': 'Default-Landscape',
 		'subtype': '2436h',
 		'scale': [ '3x' ],
-		'size': '{812, 375}'
+		'size': '{375, 812}'
```
Comment (cb1kenobi) [inline iphone/cli/commands/_build.js]: Why would a landscape orientation have a shorter width than the height? Shouldn't it be the opposite and the previous code be correct?

Comment (hansemannn) [reply iphone/cli/commands/_build.js]: I agree, but Eric reminded me that all others are the same. And native ones as well:
```
{
  "images" : [
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "2436h",
      "filename" : "Default-2436h@3x.png",
      "minimum-system-version" : "11.0",
      "orientation" : "portrait",
      "scale" : "3x"
    },
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "2436h",
      "filename" : "Default-Landscape-2436h@3x.png",
      "minimum-system-version" : "11.0",
      "orientation" : "landscape",
      "scale" : "3x"
    },
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "736h",
      "filename" : "Default-Portrait-736h@3x.png",
      "minimum-system-version" : "8.0",
      "orientation" : "portrait",
      "scale" : "3x"
    },
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "736h",
      "filename" : "Default-Landscape-736h@3x.png",
      "minimum-system-version" : "8.0",
      "orientation" : "landscape",
      "scale" : "3x"
    },
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "667h",
      "filename" : "Default-667h@2x.png",
      "minimum-system-version" : "8.0",
      "orientation" : "portrait",
      "scale" : "2x"
    },
    {
      "orientation" : "portrait",
      "idiom" : "iphone",
      "filename" : "LaunchImage-700@2x.png",
      "extent" : "full-screen",
      "minimum-system-version" : "7.0",
      "scale" : "2x"
    },
    {
      "extent" : "full-screen",
      "idiom" : "iphone",
      "subtype" : "retina4",
      "filename" : "LaunchImage-700-568h@2x.png",
      "minimum-system-version" : "7.0",
      "orientation" : "portrait",
      "scale" : "2x"
    },
    {
      "orientation" : "portrait",
      "idiom" : "ipad",
      "extent" : "full-screen",
      "minimum-system-version" : "7.0",
      "scale" : "1x"
    },
    {
      "orientation" : "landscape",
      "idiom" : "ipad",
      "extent" : "full-screen",
      "minimum-system-version" : "7.0",
      "scale" : "1x"
    },
    {
      "orientation" : "portrait",
      "idiom" : "ipad",
      "extent" : "full-screen",
      "minimum-system-version" : "7.0",
      "scale" : "2x"
    },
    {
      "orientation" : "landscape",
      "idiom" : "ipad",
      "extent" : "full-screen",
      "minimum-system-version" : "7.0",
      "scale" : "2x"
    }
  ],
  "info" : {
    "version" : 1,
    "author" : "xcode"
  }
}
```

**Pull request merged**